stages/org.osbuild.pki.update-ca-trust

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+import subprocess
+import sys
+
+import osbuild.api
+
+if __name__ == '__main__':
+    args = osbuild.api.arguments()
+    ret = subprocess.run(["chroot", args["tree"], "/usr/bin/update-ca-trust", "extract"], check=True)
```

Comment:
`check=True` will raise a `subprocess.CalledProcessError` in case of non-zero exit code. This is absolutely fine to have (it gives a lot of context) but it makes the next line a no-op (`sys.exit(ret.returncode)`).

Removing the `sys.exit` would probably be best here; and you can then drop the `sys` import as well.

Reply:
On top of that, I think that we'd want to use https://github.com/osbuild/osbuild/blob/main/osbuild/util/chroot.py for newly added code.

Reply:
Right, rebased.